src/mono/wasm/build/WasmApp.targets

```diff
@@ -52,10 +63,12 @@
     <WasmBuildNative Condition="'$(RunAOTCompilation)' == 'true'">true</WasmBuildNative>
     <WasmLinkIcalls Condition="'$(WasmLinkIcalls)' == ''">$(WasmBuildNative)</WasmLinkIcalls>
     <WasmNativeStrip Condition="'$(WasmNativeStrip)' == ''">true</WasmNativeStrip>
+    <WasmStripAOTAssemblies Condition="'$(WasmStripAOTAssemblies)' == ''">$(RunAOTCompilation)</WasmStripAOTAssemblies>
     <MonoAotCrossCompilerPath Condition="'$(MonoAotCrossCompilerPath)' == ''">$(MicrosoftNetCoreAppRuntimePackRidDir)native\cross\$(PackageRID)\mono-aot-cross</MonoAotCrossCompilerPath>
   </PropertyGroup>
 
-  <Target Name="WasmBuildApp" AfterTargets="Publish" />
+  <!-- Having this separate target allows users to cleanly add After/BeforeTargets for this -->
+  <Target Name="WasmBuildApp" AfterTargets="$(WasmBuildAppAfterThisTarget)" />
```

Comment:
Is this the topmost target to be the default target? Could we explicitly make this the default target? Would it make sense to order the targets in here in the relative order we expect them to be in?
e.g. `_BeforeWasmBuildApp` -> `_WasmResolveReferences` -> `_WasmAotCompileApp` -> etc.

Reply:
We don't want to enforce a default target. This is meant for use by user projects, and they can determine that.
When it gets packaged in some way, then this will get determined as part of that package.